app/controllers/accounts_controller.rb

```diff
@@ -524,6 +524,7 @@ def update
         end
 
         params[:account][:turnitin_host] = validated_turnitin_host(params[:account][:turnitin_host])
+        params[:account][:ally_base_url] = validated_ally_base_url(params[:account][:ally_base_url])
```

Comment:
Turnitin only requires you to enter a hostname because it hardcodes the protocol to HTTPS.
The other plugin (Equella) requires you to enter a full URL as well.

I'm a bit hesitant to make the protocol depend on the `Rails.env` object as they don't seem to do that very often.